router/router.go

```diff
@@ -239,7 +238,7 @@ func (router *Router) handleHTTPEvent(event *eventpkg.Event, w http.ResponseWrit
 		event.Data = httpdata
 		resp, err := router.callFunction(space, *backingFunction, *event)
 		if err != nil {
-			message := determineErrorMessage(err)
+			message := function.DetermineErrorMessage(err)
```

Comment:
My comment also meant that you should only use error message returned by callFunction. Not only moving the function from another package.